inc/dashboard/widget.class.php

```diff
@@ -290,7 +291,7 @@ public static function bigNumber(array $params = []): string {
          class="card big-number $class"
          title="{$p['alt']}">
          <span class="content">$formatted_number</span>
-         <div class="label">{$p['label']}</div>
```

Comment:
this change is probably not needed, no ?

Reply:
This is for when to text is shortened by the ellipsis: 
![image](https://user-images.githubusercontent.com/42734840/97280796-07ed5c80-183d-11eb-8681-e60e682b3207.png)

Reply:
sorry, i didn't see the title part